src/runtime/providers/glide.ts

```diff
@@ -0,0 +1,32 @@
+// https://glide.thephpleague.com/2.0/api/quick-reference/
+
+import { ProviderGetImage } from 'src'
+import { joinURL, encodeQueryItem, encodePath, hasProtocol } from 'ufo'
+import { createOperationsGenerator } from '~image'
+
+const operationsGenerator = createOperationsGenerator({
+  keyMap: {
+    format: 'fm',
+    fit: 'fit',
+    width: 'w',
+    height: 'h',
+    quality: 'q',
+    background: 'bg'
+  },
+  joinWith: '&',
+  formatter: (key, val) => encodeQueryItem(key, val)
+})
+
+export const getImage: ProviderGetImage = (src, { modifiers = {}, baseURL = '/' } = {}) => {
+  const params = operationsGenerator(modifiers)
+
+  if (hasProtocol(src)) {
+    return {
+      url: src
```

Comment:
Why we need this condition?

Reply:
Sorry I copied the ipx provider and forgot to remove.